openpype/tools/publisher/widgets/assets_widget.py

```diff
@@ -264,6 +264,11 @@ def reset(self, force=True):
 
         self._model.reset()
 
+    def hideEvent(self, event):
+        # Make sure to reset when we close and reopen
+        self._soft_reset_enabled = True
```

Comment:
I would say it is overkill to reset model on each show event.

Reply:
Made PR https://github.com/BigRoy/OpenPype/pull/12

Reply:
Thanks, merged it. Looks good. Would require the publisher to "reset" before it refreshes the asset list, but I suppose that's totally fine as well and will result in less database calls.